client/src/plus/icon-card/index.scss

```
@@ -10,8 +11,9 @@
   border: 1px solid var(--border-secondary);
   border-radius: var(--elem-radius);
   background-color: var(--background-primary);
-  padding: 1rem;
+  padding: 1rem 1rem 1rem 0.5rem;
```

Super nit 😄 feel free to ignore.

```suggestion
  padding: 1rem;
  padding-left: 0.5rem;
```